OtripleS.Portal.Web/Models/Users/User.cs

```diff
@@ -0,0 +1,22 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.AspNetCore.Identity;
+using System;
```

Comment:
Do run code cleanup that will order the System directives above all.

Reply:
Fixed it